### PR TITLE
fix spacing and line height issues in episode descriptions

### PR DESCRIPTION
**Pull request by mindylou**
update spacing to conform to redlined designs: 

image

### CODE REVIEW

File: Podcast/EpisodeSubjectView.swift [hunk 1]
```diff
@@ -173,7 +173,19 @@ class EpisodeSubjectView: UIView {
     func setupWithEpisode(episode: Episode) {
         episodeNameLabel.text = episode.title
         dateTimeLabel.text = episode.dateTimeLabelString
-        descriptionLabel.attributedText = episode.attributedDescription
+        // this is to avoid newlines/paragraphs showing up after truncating text
+        let stringWithoutNewlines = episode.attributedDescription.string.replacingOccurrences(of: "\n", with: "")
```
Comment (natashaarmbrust) [inline Podcast/EpisodeSubjectView.swift:177]: is this the same code as above? If so, maybe include this as an extension to NSMutableString as a function that takes in an episode

File: Podcast/PlayerEpisodeDetailView.swift [hunk 2]
```diff
@@ -102,11 +102,14 @@ class PlayerEpisodeDetailView: UIView, UIGestureRecognizerDelegate {
         episodeTitleLabel.text = episode.title
         dateLabel.text = episode.dateTimeLabelString
         let mutableString = NSMutableAttributedString(attributedString: episode.attributedDescription)
-        mutableString.addAttribute(NSAttributedStringKey.foregroundColor, value: UIColor.charcoalGrey, range: NSMakeRange(0, mutableString.length))
         let style = NSMutableParagraphStyle()
         style.lineSpacing = 4
-        mutableString.addAttribute(NSAttributedStringKey.paragraphStyle, value: style, range: NSMakeRange((0), mutableString.length))
-        mutableString.addAttribute(NSAttributedStringKey.font, value: UIFont._14RegularFont(), range: NSMakeRange(0, mutableString.length))
+        let attributes = [
```
Comment (natashaarmbrust) [inline Podcast/PlayerEpisodeDetailView.swift]: Seems like same for here so the comment above applies doubly

**Review by natashaarmbrust — CHANGES_REQUESTED** [review]
LGTM .. maybe just one refactor

File: Podcast/EpisodeSubjectView.swift (hunk 1 above)
Comment (mindylou) [reply Podcast/EpisodeSubjectView.swift:177]: done! i made an extension to `NSAttributedString`

**Pull request merged**